Sources/NIOSSL/SSLCertificate.swift

```diff
@@ -46,6 +46,16 @@ public class NIOSSLCertificate {
         case ipv4(in_addr)
         case ipv6(in6_addr)
     }
+    
+    public lazy var serialNumber: String = {
```

Comment:
Relatedly, I don't know if this property should be a `String`. A serial number is technically a number, but Swift doesn't have a currency bignum type and we don't expose one in NIOSSL. For that reason I think we should probably expose this as an array of UInt8? It's an awkward representation, but it does allow people to easily transform it into an appropriate type, or do what they need with the bytes. Our need to print it is a separate concern and we can handle that internally.